cvat-ui/src/utils/hooks.ts

```diff
@@ -122,3 +123,27 @@ export function useCardHeightHOC(params: ICardHeightHOC): () => string {
         return height;
     };
 }
+
+export function authQuery(queryParams: URLSearchParams): Record<string, string> | null {
```

Comment:
This is not a hook. I am not sure this file is a good place for this function

Reply:
Moved to utils